SPECS/libacvp/libacvp.spec

```
@@ -1,15 +1,16 @@
 Summary:        A library that implements the client-side of the ACVP protocol
 Name:           libacvp
 Version:        1.3.0
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Libraries
 URL:            https://github.com/cisco/libacvp
-# Source0:      https://github.com/cisco/%%{name}/archive/v%%{version}.tar.gz
+# Source0:        https://github.com/cisco/%%{name}/archive/v%%{version}.tar.gz
```

Since we're already here, could we remove the commented-out `Source0` and replace the line below with:

```suggestion
Source0:        https://github.com/cisco/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```